readme.md

```diff
@@ -26,7 +39,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+I do not know! My only successful form submission so far was in rails using form helpers. I don't recall using jQuery or AJAX. Please send resources to read =)
 ```
```

Comment:
Let's walk through this together! come see me during office hours and we can see it work